### DISCUSSION TITLE
Propriedade grow para Cell do componente Table

### DISCUSSION THREAD

**Discussion by vrlopess** [TiendaNube/nimbus-design-system#174] [Ideas] [answered]
Atualmente conseguimos controlar o tamanho de uma Cell através da propriedade width, porém gostaria de poder fazer isso sem definir um tamanho fixo ou uma porcentagem, que podem ficar difícil de manter proporções em certo cenários, mas usar a flexibilidade do [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow) para que o tamanho das células possam ser calculados automaticamente.

**Comment by jcampos89** [comment]
@vrlopess Atualmente só é possível fazer esse controle usando porcentagens, já que no componente table não existe essa prop. 
Você pode adicionar mais contexto sobre a issue, qual seria exatamente a dificuldade com as porcentagens?

**Reply by vrlopess** [reply, marked as answer]
@jcampos89 Perfeito! Na versão 1 eu vi que tinha a propriedade grow, mas depois vi que antes era usado Grid ao invés de Table. Como eu havia mencionado, acredito que o grow no flex conseguimos calcular melhor o tamanho e sugeri mais pra não quebrar muito a compatibilidade dos componentes que estou adaptando aqui, mas acredito que não é algo que está dando problemas no momento.
